src/forms/selectInput.test.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import SelectInput from './SelectInput';
+
+const testOptions = [
+	{ label: 'One', value: '1' },
+	{ label: 'Two', value: '2' },
+	{ label: 'Three', value: '3' }
+];
+
+describe('SelectInput basic', () => {
+	let component;
+
+	beforeEach(() => {
+		component = TestUtils.renderIntoDocument(
+			<SelectInput
+				label='Test select'
+				name='testSelect'
+				options={testOptions}
+			/>
+		);
+	});
+	afterEach(() => {
+		component = null;
+	});
+
+	it('exists', () => {
+		expect(() => TestUtils.findRenderedDOMComponentWithTag(component, 'select')).not.toThrow();
+	});
+
+	it('should have a NAME attribute', () => {
+		const selectEl = TestUtils.findRenderedDOMComponentWithTag(component, 'select');
+		expect(selectEl.name).toEqual('testSelect');
+	});
+
+	it('should have default value in state', () => {
+		expect(!!component.state.value).toBe(true);
+	});
+
+	it('default value should fall back on first option value', () => {
+		expect(component.state.value).toBe(testOptions[0].value);
+	});
+
+});
+
+describe('SelectInput advanced', () => {
+	it('should set correct value in state on change', () => {
+		const newValue = '2';
+		const onChange = jest.fn();
+		const changeSpy = spyOn(SelectInput.prototype, 'onChange').and.callThrough();
+
+		const component = TestUtils.renderIntoDocument(
+			<SelectInput
+				label='Test select'
+				name='testSelect'
+				options={testOptions}
+				onChange={onChange}
+			/>
+		);
+		const selectEl = TestUtils.findRenderedDOMComponentWithTag(component, 'select');
+
+		TestUtils.Simulate.change(selectEl, { target: { value: newValue } });
+
+		expect(changeSpy).toHaveBeenCalled();
```

Comment:
always good to confirm that the `changeSpy` has _not_ been called _before_ the simulated change event.